fedmsg.d/fmn.py

```diff
@@ -24,6 +24,8 @@
     "datanommer.sqlalchemy.url": "postgresql+psycopg2://datanommer:bunbunbun@localhost:5432/datanommer",
 
     # Some configuration for the rule processors
+    "fmn.rules.utils.use_pagure_for_ownership": False,
+    "fmn.rules.utils.pagure_api_url": "https://src.stg.fedoraproject.org/pagure/api/",
```

Comment:
The example pkgdb url given in the same file is *also* a non-prod url, so I went with that.